doc/security_audit.md

```diff
@@ -0,0 +1,6 @@
+# gRPC Security Audit
+
+A third-party security audit of gRPC C++ stack was performed by
+[Cure53](https://cure53.de) in October 2019. The full report can be found [here](https://github.com/grpc/grpc/tree/master/doc/grpc_security_audit.pdf). The medium severity issue (GRP-01-001) identified in this report was fixed in
+version 1.24.0 and above. The fix was also patched in version 1.23.1.
```

Comment:
Is the formatting off here? There seems to be a lot of space at the end of the previous line.

Reply:
Editor introduced line break. Fixed although it doesn't effect formatting.